pkg/apis/machine/v1beta1/consts.go

```diff
@@ -66,56 +48,17 @@ const (
 	// Example: cannot resolve EC2 IP address.
 	DeleteMachineError MachineStatusError = "DeleteError"
 
-	// This error indicates that the machine did not join the cluster
-	// as a new node within the expected timeframe after instance
-	// creation at the provider succeeded
-	//
-	// Example use case: A controller that deletes Machines which do
-	// not result in a Node joining the cluster within a given timeout
-	// and that are managed by a MachineSet
-	JoinClusterTimeoutMachineError = "JoinClusterTimeoutError"
-)
-
-type ClusterStatusError string
-
-const (
-	// InvalidConfigurationClusterError indicates that the cluster
-	// configuration is invalid.
-	InvalidConfigurationClusterError ClusterStatusError = "InvalidConfiguration"
-
-	// UnsupportedChangeClusterError indicates that the cluster
-	// spec has been updated in an unsupported way. That cannot be
-	// reconciled.
-	UnsupportedChangeClusterError ClusterStatusError = "UnsupportedChange"
-
-	// CreateClusterError indicates that an error was encountered
-	// when trying to create the cluster.
-	CreateClusterError ClusterStatusError = "CreateError"
-
-	// UpdateClusterError indicates that an error was encountered
-	// when trying to update the cluster.
-	UpdateClusterError ClusterStatusError = "UpdateError"
```

Comment:
Did you mean to remove these?

Reply:
Yep.
They weren't being used in MAO so I assumed it is a pretty safe bet to remove them.
Of course I could be wrong here - so looking for cloud team input.

Reply:
I think these changes should at least belong to a separate commit or a different PR. Agree - we don't use these values, we don't have and planning to have cluster resource in MAPI, yet it makes hard to approve such change.